### PR TITLE
improve course navigation styles

### PR DESCRIPTION
**Pull request by moritzlerch**
## Before:
before

***

## After:
after

***
the hover effects have also been improved 🥳

### CODE REVIEW

File: main/style.css [hunk 1]
```diff
@@ -1,3 +1,8 @@
+/* Variables */
+:root {
```
Comment (MaximilianHausen) [inline main/style.css:2]: Why did you put the variable into root and not html.dark, where all the other theme-specific colors are specified?

**Review by MaximilianHausen — COMMENTED** [review]
Looks awesome, but doesn't really work with colored tabs:
colored-headers
Also, while you're at it, you could add a newline to the end of the file.

**Comment by moritzlerch** [comment]
bruh

**Comment by moritzlerch** [comment]
Could you do the finishing touches? 🤠

**Comment by MaximilianHausen** [comment]
Okay, I'll just merge this and fix the remaining issues later

**Pull request merged**